{"name": "Успешное обновление данных авторизованного пользователя", "status": "passed", "parameters": [{"name": "field", "value": "'name'"}], "start": 1742964627606, "stop": 1742964631785, "uuid": "2db32369-4eb6-40c8-9a87-3934958811c0", "historyId": "4ed972982f3098b28b6105cfb57be880", "testCaseId": "f1d56f48b74c88041de43c56b14602ba", "fullName": "tests.test_user.TestUser#test_update_authorized_user_data_success", "labels": [{"name": "feature", "value": "Пользователь"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_user"}, {"name": "subSuite", "value": "TestUser"}, {"name": "host", "value": "LAPTOP-QVE1U26H"}, {"name": "thread", "value": "15136-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_user"}]}